CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ and this project adheres to
 - Add `test` probes as means to improve runtime test latency
   - [#4753](https://github.com/bpftrace/bpftrace/pull/4753)
 #### Changed
+- tools/opensnoop support mount point.
```

Comment:
Also there is a tools subsection this should go under.
```suggestion
- opensnoop.bt: support mount points
```

Reply:
Sorry ~